docs/manual/en/introduction/Creating-a-scene.html

```diff
@@ -144,6 +144,7 @@ <h2>The result</h2>
 			cube.rotation.y += 0.01;
 
 			renderer.render( scene, camera );
+			renderer.requestAnimationFrame(animate);
```

Comment:
The renderer uses now `renderer.setAnimationLoop( animate );` so a manual call of `requestAnimationFrame()` isn't required anymore.

Besides, `WebGLRenderer` has no `requestAnimationFrame()` method.  If anything, the line should look like so:
```js 
requestAnimationFrame(animate);
```